api/gateway/scramble.go

```diff
@@ -0,0 +1,16 @@
+package gateway
+
+import "github.com/TheThingsNetwork/ttn/utils/scramble"
+
+func (m *GPSMetadata) Scramble() (err error) {
```

Comment:
Why are gateways in Northeast Russia scrambled over a larger distance than gateways on null island?